src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/BetaDistributionShape.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.hellbender.tools.walkers.readorientation;
+
+import org.broadinstitute.hellbender.utils.Utils;
+
+public class BetaDistributionShape {
+    private double alpha;
+    private double beta;
+
+    public BetaDistributionShape(final double alpha, final double beta){
+        Utils.validateArg(alpha > 0 , "alpha must be greater than 0 but got " + beta);
```

Comment:
`ParamUtils.isPositive` is more precise.

Reply:
done